### DISCUSSION TITLE
Litlyx Widget - Open Metrics

### DISCUSSION THREAD

**Discussion by AntonioVerdiglione1996** [Litlyx/litlyx#10] [Ideas]
What do you think about a widget that can be integrated with minimal code into any website, allowing users to display their Litlyx-tracked metrics? For example, a "Page Visits" widget could elegantly show data either as text or as a text and chart combination. Each metric tracked with Litlyx could be presented as a widget.

We are open to your feedback.

I leave an image as a visual reference:
![Widget demo UI](https://github.com/user-attachments/assets/168b45ba-2199-4f5d-8993-af77b79b10b1)

**Comment by prathamjagga** [comment]
@AntonioVerdiglione1996  I think the widget will be a nice thing to do. Because I've seen websites showing the daily visits or some other metrics. For the websites already using litlyx, they wouldn't need to use any other service for this.
